py/desitarget/data/targetmask.yaml

```diff
@@ -144,6 +144,14 @@ obsmask:
     - [MORE_ZGOOD,  4, "redshift known; need more observations"]
     - [DONOTOBSERVE,  16, "Do not observe this target (possibly temporarily)"]
 
+#- Bits that can be set in TARGETID. See the schema at
+#- https://desi.lbl.gov/DocDB/cgi-bin/private/RetrieveFile?docid=2348
+targetid_mask:
+    - [BRICKID,    22, "RIGHTMOST bit that indicates BRICKID (BRICKID encompasses bits 22-41 inclusive"]
+    - [DR,         42, "RIGHTMOST bit that indicates Data Release (DR encompasses bits 42-47 inclusive"]
+    - [MOCK,       48, "this target is a mock target"]
+    - [SKY,        49, "this target is a sky fiber target"]
```

Comment:
I agree. I think it makes more sense to code this as `[NAME, FIRSTBIT, NBITS, "Description"]`. I'll make this change once we agree on the other changes.